migrate/20240314_encrypt_vm_pool_tuples.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    alter_table(:vm_pool) do
+      set_column_default(:storage_encrypted, true)
```

Comment:
I think this column shouldn't have a default value, and creator should explicitly set it. We should drop the default value. This can be done in a follow-up PR though, as this also applies to `storage_skip_sync`. So, I am okay with merging this PR as it is.